docs/user/metacat/source/install.rst

```diff
@@ -730,8 +730,10 @@ instructions for installing from source:
 
     cp <web_app_dir>/metacat <backup_dir>/metacat.<yyyymmdd>
     cp <web_app_dir>/metacat.war <backup_dir>/metacat.war.<yyyymmdd>
+    cp <web_app_dir>/metacat-index.war <backup_dir>/metacat-index.war.<yyyymmdd>
+    cp <web_app_dir>/metacatui.war <backup_dir>/metacatui.war.<yyyymmdd>
```

Comment:
Sorry for mis-communications. I wasn't talking about the backup the metacat-index.war and metacatui.war files. I was talking about the step 4, which only says copying the new metacat.war file. So we need to say copying metacat-index.war and metacatui.war as well.

Reply:
Got it! I have updated step 4, and removed the suggestion of backing up the `metacat-index.war` and `metacatui.war`